### PR TITLE
TensorFlow support, and setuptools

### PR DESCRIPTION
**Pull request by SamuelMarks**
Enjoy

### CODE REVIEW

**Comment by titu1994** [comment]
This is really impressive work ! Thanks for your contribution. 

However, the whole point of this project is to have a copy and paste-able script as the core, `se.py`, and the rest of the scripts just as helpers. With the additions of `_data/logging.yml` and conversion of this script to be more of a package, I think it has become overly heavy compared to its original purpose.

If you can revert it to its original script-like form, without the additional logging, yaml files and setup.py, I can merge this immediately. 

If you would like this to be on a seperate branch than master, thats fine too.

**Comment by SamuelMarks** [comment]
Hi @titu1994,

Yes I can make those changes, however I would like you to reconsider including setuptools support.

I'm already depending on this repository now:
https://github.com/SamuelMarks/ml-glaucoma/commit/e7f7c4651d8aa13adc213daa9a5e88ae86e67b9d#diff-ec2fb3d92db262b37034b4b1926a31e5R9
```python
from keras_squeeze_excite_network import se_resnet
```

Am confident others will find it useful in much the same way. You can even upload it to [pypi](https://pypi.org) to get interest from even more people!

PS: In terms of making the `se.py` really approachable, you can just mention it in the README with an [even] smaller snippet, and link to deeper within the repository.

**Comment by titu1994** [comment]
Hmm. Alright. Let's go with your work then. However the logging still needs to be removed. Is that alright ?

**Comment by SamuelMarks** [comment]
@titu1994 Missed your reply! - Made the changes

File: se.py [hunk 1]
```diff
@@ -1,72 +0,0 @@
-from keras.layers import GlobalAveragePooling2D, Reshape, Dense, multiply, add, Permute, Conv2D
-from keras import backend as K
-
-
-def squeeze_excite_block(input, ratio=16):
```
Comment (titu1994) [inline se.py]: This file should not be removed. While the scripts no longer depend on it, it needs to be a standalone module as it was before for easy usage I'm custom models.

**Comment by titu1994** [comment]
Thanks for the update ! Just the se.py file remains, and of you could, remove the .editorconfig file as it's no longer needed.

**Comment by titu1994** [comment]
Perfect. Ready to merge when you give the go ahead.

**Comment by titu1994** [comment]
Thanks for the fantastic PR !

**Pull request merged**